src/main/java/org/opensearch/sdk/sample/helloworld/HelloWorldExtension.java

```diff
@@ -40,7 +41,8 @@ public class HelloWorldExtension implements Extension {
     /**
      * Instantiate this extension, initializing the connection settings and REST actions.
      */
-    public HelloWorldExtension() {
+    public HelloWorldExtension() throws IOException {
+        super();
         try {
             this.settings = initializeSettings();
```

Comment:
Can we try to move all this awkward exception handling for the settings yaml (or objecct) up into the Base Extension and just have a very simple method (or methods) in the Extension itself?

Reply:
Can we either do this or create an issue to do it later?

Reply:
#182